src/components/choicesjs-stencil/interfaces.tsx

```diff
@@ -100,17 +105,23 @@ export type OnInit = () => void;
  */
 export type OnCreateTemplates = (template) => any;
 
+/**
+ * @link https://github.com/jshjohnson/Choices#valuecomparer
+ */
+export type valueCompareFunction = (value1: string, value2: string) => boolean;
```

Comment:
In general, all type declarations should be PascalCase to differentiate them from primitives